modules/installation-osp-kuryr-api-scaling.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * networking/openstack/load-balancing-openstack.adoc
+
+[id="installation-osp-kuryr-octavia-configure{context}"]
+= Scaling API using {rh-openstack} Octavia
+
+API can be scaled on {product-title} cluster running on {rh-openstack} with
+Octavia support
+Octavia is a requirement to deploy {product-title} with Kuryr, as Kuryr uses it
+to create the services, but when installing without Kuryr it is not. So first
+step is to ensure Octavia is available.
+If it is, it can be leveraged to create extra API IPs and better scale with the
+load. This allows to remove the API bottleneck as currently the traffics enters
+through the master node that holds the API VIP.
+
+If {product-title} cluster was installed with Kuryr, the Cluster Network
```

Comment:
It seems like this should be two modules, then: 
- Scaling for non-Kuryr clusters
- Scaling for Kuryr clusters

Does that seem sensible?

Reply:
Well, think is that the method is the same for with and without kuryr. With the exception that with kuryr we have one extra option (lines below: 18-21)

Reply:
@luis5tb We tend to break procedures apart based on user stories. I'll see how it looks as I edit.

Reply:
ok